### PR TITLE
Extract "following" action from "show" in DashboardsController

### PR DESCRIPTION
**Pull request by dabrorius**
## What type of PR is this? (check all applicable)

- [x] Refactor
- [ ] Feature
- [ ] Bug Fix
- [ ] Documentation Update

## Related Tickets & Documents
Currently several separate pages are all crammed under `show` action of DashboardsController.
This is the first step in an attempt of refactoring that action and separating it into several smaller controller actions.

## Added to documentation?

- [ ] docs.dev.to
- [ ] readme
- [x] no documentation needed

## [optional] What gif best describes this PR or how it makes you feel?

blek

### CODE REVIEW

**Comment by lightalloy** [comment]
Great :+1: 
I suggest adding request and feature specs for the `/dashboard/following` along with this refactoring.

**Comment by dabrorius** [comment]
@lightalloy there are request specs covering this pretty well (https://github.com/thepracticaldev/dev.to/blob/master/spec/requests/dashboard_spec.rb#L59). Do you feel there's a need for feature specs on top of this?

**Comment by lightalloy** [comment]
@dabrorius Oops, missed these specs somehow, sorry.
In my opinion, it would be nice to have feature specs on this feature. 
Maybe it's ok to have them (later) in another pr :thinking:

**Comment by dabrorius** [comment]
@lightalloy I can add feature specs as part of this PR, but I'm not sure what to test in it. I'd like to avoid adding redundant feature tests as they would slow the test suite down and would not bring any value.

What do you feel should be tested in the feature spec that is not already tested in the request spec?

**Comment by lightalloy** [comment]
@dabrorius I reviewed the existing specs again and now I agree with you that probably it's enough to have only requests specs for this functionality. However, I would add checks for the presence of the `Followed users (1)` and so on to the request specs.

File: app/views/dashboards/_actions.html.erb [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+  </a>
+  <a class="action <%= "active" if  params[:action] == "show" && (params[:which] == "user_followers" || params[:which] == "organization_user_followers") %>" href="/dashboard/user_followers">
+    <span>FOLLOWERS</span>
+    <span>(<%= @user.followers_count %>)</span>
+  </a>
+  <a class="action <%= "active" if params[:action] == "following" %>" href="/dashboard/following">
+    <span>FOLLOWING</span>
+    <span>(<%= @user.following_users_count + @user.following_tags_count %>)</span>
+  </a>
+</div>
```
Comment (lightalloy) [inline app/views/dashboards/_actions.html.erb]: A missing newline at the end of file here )

**Comment by dabrorius** [comment]
@lightalloy Makes sense, I'll expand the request specs.

**Comment by dabrorius** [comment]
@lightalloy I've added new specs as discussed. I had to re-organize the specs as rubocop started complaining that I'm nesting specs too deep.

**Comment by lightalloy** [comment]
@dabrorius Great! That was not obviously the problem of this pr, but I'm trying to make sure that we increase the test coverage when it's possible.
Thanks for your refactoring :sun_with_face:

**Pull request merged**